var/spack/repos/builtin/packages/hwloc/package.py

```diff
@@ -74,7 +74,13 @@ class Hwloc(AutotoolsPackage):
         default=(sys.platform != "darwin"),
         description="Support analyzing devices on PCI bus",
     )
-    variant("shared", default=True, description="Build shared libraries")
+    variant(
+        "libs",
+        default="shared,static",
+        values=("shared", "static"),
+        multi=True,
+        description="Build shared libs, static libs or both",
```

Comment:
Is the ability to build both libraries supported in all versions listed in this package or only from some version on?

Reply:
As far as I can tell, for all versions.

Reply:
It's more of an autotools thing, the patterns is quite common now in other libs. I'm fine with this change. Whenver a variant changes name, it's good to verify that it's not assumed to exist in things such as `setup_run_environment` etc. In this case not, so it's not a terribly breaking change to rename the variant.